test/metabase/task/analyze_queries_test.clj

```diff
@@ -23,7 +23,7 @@
 
         ;; queue the cards
         (query-analysis/with-queued-analysis
-          (run! (partial query-analysis/analyze-async! queue)
+          (run! #(query-analysis/queue-analysis! % queue)
```

Comment:
It seems like the argument order got mixed up here?

I personally find partial much easier to read, and consider it a strong idiom for closing over the "stateful" argument. I'd rather we don't have style churn here.

Reply:
No, shockingly the arg order was different between what used to be `-async` and `-sync`.

Reply:
Yikes! I just saw tons of CI exceptions and assumed it was this.

Reply:
Can we make queue come first for both? I always like to order args from longest lived / most stateful first, as this plays best with threading macros and partial.